### DISCUSSION TITLE
Text wrap for cell in tables in the current version of Grafana

### DISCUSSION THREAD

**Discussion by pyeh-seg** [grafana/grafana#62533] [Dashboards, Panels & Visualizations]
In tables panels, is there a way to text wrap within one cell? Some of the cell entries are paragraphs and it is very necessary for us.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
